src/main/java/frc/robot/subsystems/vision/VisionSubsystem.java

```diff
@@ -1,9 +1,11 @@
 package frc.robot.subsystems.vision;
 
-import edu.wpi.first.wpilibj2.command.SubsystemBase;
+public interface VisionSubsystem {
```

Comment:
To allow implementation of default methods. Making it easier to make an empty version in robotcontainer

Reply:
It doesn't get used in RobotContainer anyway.

Reply:
it is. Lines 132 to 139

Reply:
Its created on those lines, but it is never used. So I don't see why this change was necessary

Reply:
is made so that in the future the vision subsystem can be disabled without code crashing around. Or needing to fully disable functionality reliant on it.

Reply:
As far as I can tell this was accomplished by the abstract class already. Not going to hold up the PR over it though

Reply:
With a abstract class an empty version can be created. It requires a full implementation.